examples/discord_bot.py

```diff
@@ -204,34 +243,43 @@ async def current_war_status(ctx, clan_tag):
 
         e.add_field(name=war.clan.name, value=war.clan.tag)
         e.add_field(
-            name="Opponent:", value=f"{war.opponent.name}\n" f"{war.opponent.tag}", inline=False)
+            name="Opponent:",
+            value=f"{war.opponent.name}\n" f"{war.opponent.tag}", inline=False)
         e.add_field(name="War End Time:",
-                    value=f"{hours} hours {minutes} minutes {seconds} seconds", inline=False)
+                    value=f"{hours} hours {minutes} minutes {seconds} seconds",
+                    inline=False)
 
     await ctx.send(embed=e)
 
 
-async def run_tests_and_quit():
-    # ignore this; it is purely for the benefit of being able to run the examples as tests.
-    import sys
-
-    class Msg:
-        _state = bot._connection
+async def main():
+    logging.basicConfig(level=logging.ERROR)
 
-    await bot.wait_until_ready()
-    ctx = commands.Context(prefix=None, message=Msg, bot=bot)
+    # Instantiate the coc client used to log in
+    coc_client = coc.EventsClient(
+        key_names="coc.py bot example"
+    )
 
-    async def _mock_get_channel():
-        return bot.get_channel(INFO_CHANNEL_ID)
-    ctx._get_channel = _mock_get_channel
+    # Attempt to log into CoC API using your credentials.
+    try:
+        await coc_client.login(os.environ.get("DEV_SITE_EMAIL"),
+                               os.environ.get("DEV_SITE_PASSWORD"))
+    except coc.InvalidCredentials:
```

Comment:
Will this actually raise invalid credentials? Not sure what that `raise` line does back in Client.login